webrender/src/device/gfx/device.rs

```diff
@@ -545,19 +556,15 @@ impl<B: hal::Backend> Device<B> {
             Vec::new(),
         );
 
-        let locals_descriptors = if cfg!(feature = "push_constants") {
-            None
-        } else {
-            Some(DescriptorSetHandler::new(
-                &device,
-                &mut desc_allocator,
-                &descriptor_data,
-                &DescriptorGroup::Default,
-                DESCRIPTOR_SET_LOCALS,
-                descriptor_count.unwrap_or(DESCRIPTOR_COUNT),
-                Vec::new(),
-            ))
-        };
+        let locals_descriptors = DescriptorSetHandler::new(
+            &device,
+            &mut desc_allocator,
+            &descriptor_data,
+            &DescriptorGroup::Default,
+            DESCRIPTOR_SET_LOCALS,
+            descriptor_count.unwrap_or(DESCRIPTOR_COUNT),
```

Comment:
I think we should just use 1 descriptor if we have push_constant support:
`if use_push_consts { 1 } else { descriptor_count.unwrap_or(DESCRIPTOR_COUNT) }`